code/game/response_team.dm

```diff
@@ -22,137 +12,83 @@ var/can_call_ert
-		var/index = security_state.all_security_levels.Find(security_state.current_security_level)
-		ert_base_chance += 2**index
-		sleep(600 * 3) // Minute * Number of Minutes
-
-
-proc/trigger_armed_response_team(var/force = 0)
-	if(!can_call_ert && !force)
-		return
-	if(send_emergency_team)
+		for(var/datum/emergency_call/C in ticker.mode.all_calls)
+			if(C.name == choice)
+				ticker.mode.picked_call = C
+				break
+
+	if(!istype(ticker.mode.picked_call))
 		return
-	if(!current_ert)
-		current_ert = pick(ert_options)
-	var/send_team_chance = ert_base_chance // Is incremented by increment_ert_chance.
-	send_team_chance += 2*percentage_dead() // the more people are dead, the higher the chance
-	send_team_chance += percentage_antagonists() // the more antagonists, the higher the chance
-	send_team_chance = min(send_team_chance, 100)
-
-	if(force) send_team_chance = 100
-
-	// there's only a certain chance a team will be sent
-	if(!prob(send_team_chance))
-		command_announcement.Announce("It would appear that an emergency response team was requested for [station_name()]. Unfortunately, we were unable to send one at this time.", "[GLOB.using_map.boss_name]")
-		can_call_ert = 0 // Only one call per round, ladies.
+
+	var/max = input("What should the maximum amount of mobs be?", "Max Mobs", ticker.mode.picked_call.mob_max) as null|num
+	if(!max || max < 1)
 		return
 
-	command_announcement.Announce("It would appear that an emergency response team was requested for [station_name()]. We will prepare and send one as soon as possible.", "[GLOB.using_map.boss_name]")
-	evacuation_controller.add_can_call_predicate(new/datum/evacuation_predicate/ert())
+	ticker.mode.picked_call.mob_max = max
+
+	var/min = input("What should the minimum amount of mobs be?", "Min Mobs", ticker.mode.picked_call.mob_min) as null|num
+	if(!min || min < 1)
+		min = 0
 
-	can_call_ert = 0 // Only one call per round, gentleman.
-	send_emergency_team = 1
+	ticker.mode.picked_call.mob_min = min
 
-	sleep(3 MINUTES)
-	send_emergency_team = 0 // Can no longer join the ERT.
-	current_ert.pick_candidates()
+	var/is_announcing = TRUE
+	if(alert(usr, "Would you like to announce the distress beacon to the server population? This will reveal the distress beacon to all players.", "Announce distress beacon?", "Yes", "No") != "Yes")
+		is_announcing = FALSE
 
+	ticker.mode.picked_call.activate(is_announcing)
 
-/datum/evacuation_predicate/ert
-	var/prevent_until
+	log_admin("[key_name(usr)] called a [choice == "Randomize" ? "randomized ":""]distress beacon: [ticker.mode.picked_call.name]. Min: [min], Max: [max].")
+	message_admins("[key_name(usr)] called a [choice == "Randomize" ? "randomized ":""]distress beacon: [ticker.mode.picked_call.name] Min: [min], Max: [max].")
 
-/datum/evacuation_predicate/ert/New()
-	..()
-	prevent_until = world.time + 30 MINUTES
 
-/datum/evacuation_predicate/ert/is_valid()
-	return world.time < prevent_until
+/client/verb/JoinResponseTeam()
```

Comment:
ah i see you just moved this.

Honestly though, remove it. Antag settings are enough opting in, we dont need people to manually click a verb every round

Reply:
This verb is still here, this isn't resolved